front/app/modules/free/user_confirmation/index.tsx

```diff
@@ -13,7 +13,7 @@ const configuration: ModuleConfiguration = {
         <ConfirmationSignupStep {...props} />
       </FeatureFlag>
     ),
-    'app.containers.Admin.settings.registrationBeginning': (props) => (
+    'app.containers.Admin.settings.registrationSectionEnd': (props) => (
```

Comment:
Important: outlets are places in the UI where we can insert components. Naming them well is important. We should have some conventions for this. I'll post a message soon.